### PR TITLE
CONCF-673 Use 'standard' video object to render trending videos

### PR DESCRIPTION
**Pull request by hakubo**

### CODE REVIEW

File: front/scripts/main/components/TrendingVideosItemComponent.ts [hunk 1]
```diff
@@ -8,41 +8,38 @@ App.TrendingVideosItemComponent = Em.Component.extend(App.ViewportMixin, App.Tra
...
 				height: this.get('imageHeight'),
 				mode: this.get('cropMode')
-			},
-			imageUrl: string = this.thumbnailer.getThumbURL(this.get('imageUrl'), options);
+			};
 
-		this.set('currentlyRenderedImageUrl', imageUrl);
-	},
+		if (this.get('video.url')) {
+			return this.thumbnailer.getThumbURL(this.get('video.url'), options);
```
Comment (rogatty) [inline front/scripts/main/components/TrendingVideosItemComponent.ts]: How about `var videoUrl = this.get('video.url')`?

File: front/templates/main/components/main-page.hbs [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
-	{{/if}}
-	{{#if model.trendingArticles}}
-		{{trending-articles model=model.trendingArticles}}
-	{{/if}}
-	{{#if model.trendingVideos}}
-		{{trending-videos model=model.trendingVideos}}
-	{{/if}}
-</section>
+{{#if model.wikiaStats}}
+    {{wikia-stats model=model.wikiaStats}}
```
Comment (rogatty) [inline front/templates/main/components/main-page.hbs]: We use tabs in .hbs files.

**Comment by rogatty** [comment]
Small things and LGTM

**Comment by bognix** [comment]
after minor fixes LGTM

File: front/templates/main/components/main-page.hbs (hunk 2 above)
Comment (hakubo) [inline front/templates/main/components/main-page.hbs]: fixed in CONCF-674

File: front/scripts/main/components/TrendingVideosItemComponent.ts (hunk 1 above)
Comment (hakubo) [inline front/scripts/main/components/TrendingVideosItemComponent.ts]: :+1:

**Comment by rogatty** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM

**Comment by bognix** [comment]
ping @hakubo

**Comment by bognix** [comment]
LGTM

**Comment by vforge** [comment]
:+1: LGTM

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**